### ISSUE TITLE
Regarding open review, if done again, I would NEVER....

### ISSUE THREAD

**Issue by timm** [researchart/icsme16#20]

**Comment by grammarware** [comment]
The process would probably not scale to a much larger evaluation procedure with a few dozen artefacts, the authors and reviewers of each would have all kinds of rights for the repo and would get all sorts of notifications by email.

**Comment by timm** [comment]
right... one notification for every issue on every paper, over the whole cycle

hmmm....

it could be that we did things at the wrong granularity. it should have been one organization for the track and one repo per paper within that track

**Comment by emhill** [comment]
I think changing the granularity would address that communication issue nicely. The next question is... can you automate the process easily? :-)

**Comment by timm** [comment]
git has a whole api for doing stuff. you need to get a magic token to run those scripts (see https://gist.github.com/timm/a87fff1d8f0210372f26#file-gitable-py-L56) otherwise you get blocked as a DOS attack. but getting that token is trivial:
- In terminal run the following command
  
    curl -i -u <your_username> -d '{"scopes": ["repo", "user"], "note": "OpenSciences"}' https://api.github.com/authorizations
- Enter ur password on prompt. You will get a JSON response. 

In that response there will be a key called "token" . 

Note that after that, the scripts could be kept in the github  repo and the token stored in some file in a super-super directory outside of git
